internal/cli/internal/command/image/update.go

```diff
@@ -0,0 +1,125 @@
+package image
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/superfly/flyctl/pkg/iostreams"
+
+	"github.com/superfly/flyctl/api"
+	"github.com/superfly/flyctl/internal/cli/internal/app"
+	"github.com/superfly/flyctl/internal/cli/internal/command"
+	"github.com/superfly/flyctl/internal/cli/internal/flag"
+	"github.com/superfly/flyctl/internal/cli/internal/prompt"
+	"github.com/superfly/flyctl/internal/cli/internal/render"
+	"github.com/superfly/flyctl/internal/cli/internal/watch"
+	"github.com/superfly/flyctl/internal/client"
+)
+
+func newUpdate() *cobra.Command {
+	const (
+		long = `This will update the application's image to the latest available version.
+The update will perform a rolling restart against each VM, which may result in a brief service disruption.
+		`
+		short = "Updates the app's image to the latest available version. (Fly Postgres only)"
+
+		usage = "update"
+	)
+
+	cmd := command.New(usage, short, long, runUpdate,
+		command.RequireSession,
+		command.RequireAppName,
+	)
+
+	flag.Add(
+		cmd,
+		flag.App(),
+		flag.AppConfig(),
+		flag.Bool{
+			Name:        "detach",
+			Description: "Return immediately instead of monitoring update progress",
+		},
+	)
+
+	return cmd
+}
+
+func runUpdate(ctx context.Context) error {
+	var (
+		client  = client.FromContext(ctx).API()
+		io      = iostreams.FromContext(ctx)
+		appName = app.NameFromContext(ctx)
+	)
+
+	app, err := client.GetImageInfo(ctx, appName)
+	if err != nil {
+		return err
```

Comment:
there should be a decoration here: `fmt.Errorf("failed retrieving image info: %w", err)`